domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -268,6 +279,34 @@ constructor(
         useCaseGroup = useCaseGroupBuilder.build()
     }
 
+    @OptIn(ExperimentalCamera2Interop::class)
+    private fun createPreviewUseCase() : Preview {
+        var isPreviewStabilizationSupported = false
+
+        // TODO: find a better way to access camera info of front/back camera before binding
+        //  use cases
+        val availableCameraInfo = cameraProvider.availableCameraInfos
+        if (availableCameraInfo.isNotEmpty()) {
+            for (cameraInfo in availableCameraInfo) {
+                val lensFacing = Camera2CameraInfo.from(cameraInfo).getCameraCharacteristic(
+                    CameraCharacteristics.LENS_FACING)
+                if ((isFrontFacing && CameraCharacteristics.LENS_FACING_FRONT == lensFacing)
+                    || (!isFrontFacing && CameraCharacteristics.LENS_FACING_BACK == lensFacing)) {
+                    isPreviewStabilizationSupported =
+                        Preview.getPreviewCapabilities(cameraInfo)
```

Comment:
Looks like `Preview.getPreviewCapabilities()` is restricted. You'll need to suppress the warning and we should track updating this when it is no longer restricted.

Reply:
The suppress warning is because of Camera2CameraInfo, so this is not necessary now after using `CameraSelector.filter`.

Reply:
Android studio still gives me a red squiggly under `getPreviewCapabilities()`, so you may need `@SuppressLint("RestrictedApi")` to suppress it.

Reply:
I unhide this API several days ago, aosp/2786388.
Are you using the old build id snapshot?

Reply:
Ah it looks like my project was still syncing :)
Looks good now.

Reply:
Thanks!